### PR TITLE
Two new video CC events, two new common fields

### PR DESCRIPTION
**Pull request by lamagnifica**
@pmitros, @mulby, please review for completeness and accuracy.
@stroilova FYI. @mhoeber @catong, this is in today’s release.

### CODE REVIEW

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 1]
```diff
@@ -150,7 +154,16 @@ Common Fields
...
+definitions of all events. These fields are at the root level of the event
+JSON documents.
+
+===========================
+``accept_language`` Field
+===========================
+
+**Type:** string
+
+**Details:** The value from the HTTP Accept-Language request-header field.
```
Comment (catong) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: Accept_Language (underscore rather than hyphen)?

**Comment by catong** [comment]
Just a couple of questions otherwise :+1:

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 2]
```diff
@@ -1174,6 +1199,19 @@ Mobile App Event`.
        ``courseware_studentmodule.module_id``. For example, 
        ``i4x-HarvardX-PH207x-video-Simple_Random_Sample``.
 
+``video_hide_cc_menu`` and ``video_show_cc_menu``
+**************************************************
+
+The browser emits a ``video_show_cc_menu`` when a user selects **CC** in the
```
Comment (catong) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: Is "a" (the article) needed here? I find it slightly strange without a noun following the event name, but I'm not sure what your standard wording is. Could you just say "The browser emits `video_show_cc_menu` when a user selects..."?

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 3]
```diff
@@ -1174,6 +1199,19 @@ Mobile App Event`.
...
+video player to show the transcript or menu of language options. The browser
+emits a ``video_hide_cc_menu`` when a user selects **CC** again to close the
```
Comment (catong) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: Same comment as above

File: en_us/data/source/internal_data_formats/tracking_logs.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: I looked it up on our new partner, the W3C's, web site to make sure I got it right: http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html

File: en_us/data/source/internal_data_formats/tracking_logs.rst (hunk 2 above)
Comment (lamagnifica) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: missed the word "event", then copied and pasted... thanks!

**Comment by pmitros** [comment]
:+1: 

Not a blocker, but if it were me, I'd add links to: 
http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.4
http://en.wikipedia.org/wiki/HTTP_referer

**Comment by mulby** [comment]
I second the comment about the link. I was confused about the misspelling until I read that article. Agreed it's not a blocker.

:+1:

**Comment by lamagnifica** [comment]
Added xrefs to the W3C spec and also the history info that I forgot earlier for the two new common fields.

**Comment by stroilova** [comment]
Thank you @lamagnifica .

**Comment by stroilova** [comment]
By the way, @pmitros  if this has been merged in to the release, would you like me to help validate the events are going through and match these docs in production? I would be happy to help with Splunk access. Thanks.

**Comment by pmitros** [comment]
@stroilova Gabe helped me do this with last push to stage.

**Pull request merged**